.gitlab/includes/clang14_cuda11_pipeline.yml

```diff
@@ -28,25 +28,25 @@ include:
 clang14_cuda11_spack_compiler_image:
   extends:
     - .variables_clang14_cuda11_config
-    - .compiler_image_template_rosa
+    - .compiler_image_template_gh200
 
 clang14_cuda11_spack_image:
   needs: [clang14_cuda11_spack_compiler_image]
   extends:
     - .variables_clang14_cuda11_config
-    - .dependencies_image_template_rosa
+    - .dependencies_image_template_gh200
 
 clang14_cuda11_build:
   needs: [clang14_cuda11_spack_image]
   extends:
     - .variables_clang14_cuda11_config
-    - .build_template_rosa
+    - .build_template_gh200
 
 .clang14_cuda11_test_common:
   needs: [clang14_cuda11_build]
   extends:
     - .variables_clang14_cuda11_config
-    - .test_common_gpu_daint_cuda
+    - .test_common_gh200
```

Comment:
Not 100% sure, since the cluster name might change and we don't really care which one it runs on as long as it has Grace-Hopper? We do need to mention the cluster name of course in the base templates `*_gh200`, but I'm not sure (yet) that the exact cluster is a relevant distinction for the different configurations. If we find out that it is worth running pipelines on different gh200 clusters we can introduce the distinction at that point? What do you think?

Reply:
I followed your advice in the end, since we call the other test template `test_common_eiger_mc`, i.e. we include the cluster name there. So I've included `daint` in the Grace-Hopper template as well.